wiki/creeper-spores.md

```diff
@@ -4,6 +4,8 @@ slug: creeper-spores
 layout: wiki
 ---
 
+*[![img](https://img.shields.io/discord/292744693803122688?color=informational&label=Ladysnake&logo=Discord)](https://ladysnake.glitch.me)[![img](http://cf.way2muchnoise.eu/full_creeper-spores_downloads.svg)](https://www.curseforge.com/minecraft/mc-mods/creeper-spores)[![img](http://cf.way2muchnoise.eu/versions/minecraft_creeper-spores_latest.svg)](https://www.curseforge.com/minecraft/mc-mods/creeper-spores)[![img](https://img.shields.io/github/last-commit/ladysnake/creeper-spores)](https://github.com/ladysnake/creeper-spores/commits/main)*
```

Comment:
I don't think there is much of a point in putting the discord badge here, it's already in the website's header. Also the "last commit" badge is probably not too useful either ?